specification/compute/resource-manager/Microsoft.Compute/stable/2019-04-01/skus.json

```diff
@@ -54,7 +54,7 @@
             "in": "query",
             "required": false,
             "type": "string",
-            "description": "The filter to apply on the operation."
+            "description": "The filter to apply on the operation. Only **location** filter is supported currently."
```

Comment:
Can you give an example?

Reply:
Example was already added as part of a previous PR: https://github.com/Azure/azure-rest-api-specs/pull/7167/files